src/glmtools.jl

```diff
@@ -11,41 +11,122 @@ type LogLink <: Link end
 type ProbitLink <: Link end
 type SqrtLink <: Link end
 
+"""
+Link function mapping mu to eta, the linear predictor (CauchitLink response model).
```

Comment:
Thanks. But instead of repeating the description of the function for each method, you'd better document the function itself only once:
```julia
"""
Link function mapping μ to η (the linear predictor).
"""
function linkfun end
```

Then I don't think you need to document the individual methods, as their signature is explicit enough.

Also, using the greek letters sounds better to me than spelling them in full.